pkg/conthandler/container_aggregator_interface.go

```diff
@@ -0,0 +1,7 @@
+package conthandler
+
+type ContainerAggregatorClient interface {
+	StartAggregate() error
+	StopAggregate() error
+	GetContainerRealtimeFileList() []string
```

Comment:
```suggestion
	ListContainerRealTimeFiles() []string
```
How can you have the `Get` and `List` in a single function name? Is it `Get` or `List`?